### ISSUE TITLE
Scroll Bar on Columns.

### ISSUE THREAD

**Issue by ham1204** [tnichols217/obsidian-columns#36]
When two columns are lined up, if one is longer than the other, 
I would like to be able to limit the height of that column and add a scroll bar.

**Comment by tnichols217** [comment]
Limit to the height of the shortest column? Or a custom height?

**Comment by ham1204** [comment]
Limit to the height for longer column. or custom height for each one... is it difficult?

**Comment by ham1204** [comment]
Because I use a lot of note embedding with your plugin and sometimes one note I embed is too long. (right column exp.)
<img width="621" alt="SCR-20221112-ieh-2" src="https://user-images.githubusercontent.com/90486071/201456723-fc4b506c-44ec-42ee-94f5-8d2c43ab2389.png">

**Comment by tnichols217** [comment]
fixed in v1.3.0 commit https://github.com/tnichols217/obsidian-columns/commit/77d3b0936c7ff7418f2ec9f30472ed3ec1d61ffb

Instructions are in the readme for how to use :)

`````
````col
height=shortest
===
```col-md
line 1
line 2
line 3
line 4
```

```col-md
line 1
line 2
```
````
`````
